docs/source/en/quicktour.mdx

```diff
@@ -14,83 +14,57 @@ specific language governing permissions and limitations under the License.
+* Build a custom model by modifying the base configuration, preprocessing, and model classes.
+* Train a model with the [`Trainer`] class, an optimized training loop for 🤗 Transformers models.
 
-All code examples presented in the documentation have a toggle on the top left for PyTorch and TensorFlow. If
-not, the code is expected to work for both backends without any change.
+Whether you're a new user or an experienced developer, this quick tour will help you get started and show you the most important features. If you're a beginner, we recommend starting with our tutorials or the [Hugging Face course](https://huggingface.co/course/chapter1/1) for a more in-depth introduction.
 
-</Tip>
+Let's take a look at each of these features and get up and running!
 
-## Pipeline
+Before you begin, make sure you have 🤗 Transformers installed:
 
-[`pipeline`] is the easiest way to use a pretrained model for a given task.
+```bash
+pip install transformers
+```
 
-<Youtube id="tiZFewofSLM"/>
+You'll also want to install 🤗 Datasets to load a dataset to train on quickly:
 
-The [`pipeline`] supports many common tasks out-of-the-box:
+```bash
+pip install datasets
+```
 
-**Text**:
-* Sentiment analysis: classify the polarity of a given text.
-* Text generation (in English): generate text from a given input.
-* Name entity recognition (NER): label each word with the entity it represents (person, date, location, etc.).
-* Question answering: extract the answer from the context, given some context and a question.
-* Fill-mask: fill in the blank given a text with masked words.
-* Summarization: generate a summary of a long sequence of text or document.
-* Translation: translate text into another language.
-* Feature extraction: create a tensor representation of the text.
+## Pipeline
 
-**Image**:
-* Image classification: classify an image.
-* Image segmentation: classify every pixel in an image.
-* Object detection: detect objects within an image.
+The [`pipeline`] is the easiest way to use a pretrained model for inference. You can use the [`pipeline`] out-of-the-box for many tasks such as text generation, image classification, automatic speech recognition, and many more. 
 
-**Audio**:
-* Audio classification: assign a label to a given segment of audio.
-* Automatic speech recognition (ASR): transcribe audio data into text.
+<Youtube id="tiZFewofSLM"/>
 
 <Tip>
 
-For more details about the [`pipeline`] and associated tasks, refer to the documentation [here](./main_classes/pipelines).
+For more details about tasks supported by [`pipeline`], take a look at the [pipelines API reference](./main_classes/pipelines).
 
 </Tip>
 
-### Pipeline usage
-
-In the following example, you will use the [`pipeline`] for sentiment analysis.
-
-Install the following dependencies if you haven't already:
-
-<frameworkcontent>
-<pt>
-```bash
-pip install torch
-```
-</pt>
-<tf>
-```bash
-pip install tensorflow
-```
-</tf>
-</frameworkcontent>
```

Comment:
The code won't work if neither PyTorch or TensorFlow is installed, so this needs to stay in some way.